hoot-services/src/main/java/hoot/services/controllers/grail/DeriveChangesetReplacementCommand.java

```diff
@@ -89,9 +128,9 @@ public static int exponent(Double d) {
      * (width or height) in decimal degrees
      */
     public static int adaptiveAlpha(Double d) {
-        int exp = 4 + exponent(d); //follows table here https://en.wikipedia.org/wiki/Decimal_degrees
+        int exp = 5 + exponent(d); //follows table here https://en.wikipedia.org/wiki/Decimal_degrees
```

Comment:
So I tweaked this again to hopefully get an alpha distance to avoid the `Alpha shape area is zero` error.